bcipy/acquisition/demo/demo_client_and_server.py

```diff
@@ -19,49 +19,60 @@ def main():
     sys.path.append('..')
     sys.path.append('../..')
 
-    from bcipy.acquisition.datastream import generator
     from bcipy.acquisition.protocols import registry
     from bcipy.acquisition.client import DataAcquisitionClient
-    from bcipy.acquisition.datastream.server import DataServer
+    from bcipy.acquisition.datastream.tcp_server import TcpDataServer
+    from bcipy.acquisition.devices import supported_device
+    from bcipy.acquisition.connection_method import ConnectionMethod
+    from bcipy.helpers.system_utils import log_to_stdout
+
+    if debug:
+        log_to_stdout()
 
     host = '127.0.0.1'
     port = 9000
     # The Protocol is for mocking data.
```

Comment:
Comment is a little bit unclear - is it to mock the data or is it using the mocked data?

Reply:
I refactored this code but forgot to remove this comment. It's not relevant anymore so I'll delete it. Sorry for the confusion.